src/matrix_utilities.jl

```diff
@@ -11,14 +11,17 @@ lsqrt(m::AbstractMatrix) = cholesky(m).L
 lsqrt(m::UniformScaling) = sqrt(m)
 
 # stein operator
-stein(Σ::AbstractMatrix, Φ::AbstractMatrix) = Matrix(Hermitian(Φ * Σ * Φ'))
-stein(Σ::AbstractMatrix, Φ::AbstractMatrix, Q::AbstractMatrix) =
-    Matrix(Hermitian(Φ * Σ * Φ' + Q))
-stein(Σ::AbstractMatrix, A::AbstractAffineMap) = stein(Σ, slope(A))
-stein(Σ::AbstractMatrix, A::AbstractAffineMap, Q::AbstractMatrix) = stein(Σ, slope(A), Q)
+stein(Σ, Φ) = Matrix(Hermitian(Φ * Σ * Φ'))
```

Comment:
Why the `Matrix` around it?

Reply:
I do recall there being some reason for enforcing covariance matrices to be Hermitian without putting them in a Hermitian view, dont recall which one though. So am open to the idea that its not necessary.

Reply:
Lets fix this later.